# DP: gm2: use time_t rather than long as a cast

gcc/gm2/

2019-10-10      Gaius Mulley <gaius.mulley@southwales.ac.uk>

	* gm2/m2.flex:  use time_t rather than long as the
          parameter type casted to 0.

--- a/src/gcc/gm2/m2.flex
+++ b/src/gcc/gm2/m2.flex
@@ -351,10 +351,10 @@ static void handleColumn (void)
 
 static void handleDate (void)
 {
-  time_t  clock = time((long *)0);
-  char   *sdate = ctime(&clock);
-  char   *s     = (char *)alloca(strlen(sdate)+2+1);
-  char   *p     = index(sdate, '\n');
+  time_t  clock = time ((time_t *)0);
+  char   *sdate = ctime (&clock);
+  char   *s     = (char *) alloca (strlen (sdate) + 2 + 1);
+  char   *p     = index (sdate, '\n');
 
   if (p != NULL) {
     *p = (char) 0;
